python_modules/libraries/dagster-aws/dagster_aws/ecs/container_context.py

```diff
@@ -297,6 +325,7 @@ def create_for_run(dagster_run: DagsterRun, run_launcher: Optional["EcsRunLaunch
                     mount_points=run_launcher.mount_points,
                     volumes=run_launcher.volumes,
                     run_sidecar_containers=run_launcher.run_sidecar_containers,
+                    run_ecs_tags=run_launcher.run_ecs_tags,
```

Comment:
this EcsContainerContext.create_for_run method is called within the run launcher to do the merge i mentioned. First it creates an EcsContainerContext that just uses the instance-level / run launcher config, then it merges it with whatever code-location-specific information was stored on the run (see the 'run_container_context' value below)

They are merged together and create a final EcsContainerContext to be used to launch the run